core/server/worker/src/main/java/alluxio/Sessions.java

```diff
@@ -36,6 +38,9 @@
   public static final int ACCESS_BLOCK_SESSION_ID = -5;
   public static final int KEYVALUE_SESSION_ID = -6;
 
+  // internal session id base should be smaller than all predefined session ids
```

Comment:
does this mean that after this PR, we can no longer add new types of session IDs without breaking backwards compatibility?

Reply:
I think we can still add new predefined session IDs - just bump up the internal session id base to give space to the new ID. It will not break compatibility as we do not persist sessionID.

Reply:
I see - this isn't so much of an issue because it doesn't affect clients, and usually all of our servers are running the same version.

Still, I think we can easily save ourselves some trouble later by bumping this to `-100` or `-1000`. Then we don't need to remember to bump the internal session id base, and we don't break rolling upgrade scenarios